domain/metadata/transaction_metadata.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from domain.metadata.metadata import Metadata, MetadataType
+
+
+@dataclass
+class MetaTransaction:
```

Comment:
If we add any new property in transaction metadata we need to update the explorer service? Imagine we want to mark a tx as banned or as important (don't know why but just imagining possible situations). How do we handle this?